backend/src/libs/packages/database/schema/tables-schema.ts

```diff
@@ -60,4 +63,51 @@ const business = pgTable('business_details', {
   updatedAt: timestamp('updated_at').notNull().defaultNow(),
 });
 
-export { business, groups, users, usersRelations };
+const businessRelations = relations(users, ({ many }) => ({
+  orders: many(orders),
+}));
+
+const orderStatuses = Object.values(OrderStatus as Record<number, string>) as [
+  string,
+  ...string[],
+];
+
+const statusEnum = pgEnum('status_enum', orderStatuses);
```

Comment:
let's create helper for this and keep our schemas file clean

Reply:
done